domains/marioannier.json

```diff
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "Marioagp",
+        "email": "mario.annier@gmail.com"
+      },
+
+      "record": {
+        "A": ["marioannier"]
```

Comment:
Invalid A record. Must be valid IPv4.

Reply:
I've changed the code in marioannier.json for:
```
{
    "owner": {
        "username": "marioannier",
        "email": "mario.annier@gmail.com"
    },
    "record": {
        "A": ["217.174.245.249"],
        "MX": ["hosts.is-a.dev"],
        "TXT": "v=spf1 a mx ip4:217.174.245.249 ~all"
    }
}
```
I don't know if this solves the problem.